chapter3.adoc

```diff
@@ -33,9 +34,11 @@ an illegal instruction exception.
 [wavedrom, ,svg]
 ....
 {reg: [
-  {bits:  44, name: 'MTTPPN'},
-  {bits:  16, name: 'SDID'},
+  {bits:  6, name: 'SDID'},
+  {bits:  2, name: 'Zero'},
   {bits:  4, name: 'Mode'},
+  {bits:  44, name: 'MTTPPN'},
+  {bits:  8, name: 'Zero'},
```

Comment:
Zero -> WPRI
MTTPPN -> PPN (since the CSR already says its MTT).

Reply:
@kasanovic any feedback on this change - please ack if none, then I can merge it.

Reply:
Should we organize the fields as: `[MODE | WPRI | PPN]`. Instead of splitting the reserved field into two group of bits make it one contiguous run of reserved bits between the `MODE` and `PPN` fields.

Reply:
so the first WPRI bits are for other control bits if needed (like lock) and the second WPRI bits are for the PPN expansion (future)

Reply:
The other controls could be filled in the WPRI run of bits for future - growing towards right. The PPN would want to expand and grow towards left.

Reply:
ack